### PR TITLE
Cart and Wishlist pages

### PR DESCRIPTION
**Pull request by samyukthasudhakar**
As part of this PR the following pages have been built :

- Cart Page : https://cart-and-wishlist.netlify.app/cart.html
- Wishlist Page : https://cart-and-wishlist.netlify.app/wishlist.html

Let me know if any changes are required. Thank you !

### CODE REVIEW

**Review by ParthParolekar — APPROVED** [review]
Everything looks great.

**Review by partha8 — APPROVED** [review]
LGTM

File: styles/cart.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    width:2rem;
+    height:2rem;
+    font-size: medium;
+}
+
+.cart-card-dismiss:hover{
+    border:none;
+    border-radius:50%;
+    background-color: var(--secondary-clr);
+}
```
Comment (ApurvaSawant11) [inline styles/cart.css:52]: This hover is not working for purple-colored cards.

**Review by ApurvaSawant11 — COMMENTED** [review]
Everything looks good!

**Review by ApurvaSawant11 — APPROVED** [review]
Approved!

File: styles/cart.css (hunk 1 above)
Comment (samyukthasudhakar) [reply styles/cart.css:52]: Since the color on hover is same as the card color it's not being shown. Will change that to a contrast color. Thanks.

**Pull request merged**